lib/beaker/hypervisor/aws_sdk.rb

```diff
@@ -405,7 +400,12 @@ def launch_all_nodes
           instance = create_instance(host, ami_spec, nil)
           instances.push({:instance => instance, :host => host})
         end
+        # Add metadata tags to each instance
```

Comment:
I think that if you can retarget your work to the [beaker-aws](https://github.com/puppetlabs/beaker-aws) repo, we can merge your work in after the initial release is out the door. We definitely don't need to discover all the gotchas ourselves, so I hope that you can target your work there and we can merge in your changes soon.